src/client/QXmppPubSubManager.h

```diff
@@ -20,13 +20,19 @@
 class QXmppPubSubPublishOptions;
 class QXmppPubSubSubscribeOptions;
 
-extern const char *PUBSUB_CURRENT_ITEM;
-
 class QXMPP_EXPORT QXmppPubSubManager : public QXmppClientExtension
 {
     Q_OBJECT
 
 public:
+    enum ItemId {
+        Current  ///< Item of a singleton node
+    };
+
+    const QVector<QString> ITEM_IDS = {
+        QStringLiteral("current")
+    };
+
```

Comment:
this shouldn't be public and if you add it here it needs to be static (otherwise it would affect the size of QXmppPubSubManager)

I'd suggest to do:
```
private:
    static QString standardItemIdToString(StandardItemId);
```
(and then defining it in the .cpp file)